recipes/grad-cam/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "grad-cam" %}
+{% set version = "1.3.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/grad-cam-{{ version }}.tar.gz
+  sha256: 26e8f0f2e0e28e7c4f486f26886ebd43f1ecff58a015cf065f8f697551944c80
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - libgdal
+  run:
+    - python >=3.6
+    - opencv >=4.1
+    - pytorch >=1.7.0
+    - torchvision >=0.8.2
+    - tqdm >=4.42
+    - ttach >=0.0.3
+    - typing-extensions ==3.7.4.3
+    - pillow
```

Comment:
Pillow isn't listed in `setup.py`, and only seems to be referenced in `requirements.txt`, but not in the code. Are you sure it's needed?

Reply:
I tried running the recipe without Pillow and it ran just fine. So, thank you, for pointing this out. However, I have some reservations against removing `pillow` from the recipe for the follow reasons:

- I don't maintain `grad-cam` and building/maintaining the library is relatively convenient if I follow the specifications made in `requirements.txt`.
- In case of some manual error, if there is a difference between `setup.py` and `requirements.txt`, I would prefer to include the superset (union) of the two. This will atleast ensure no functionality is broken.
- However, while doing so, I would also prefer to raise an issue with the library's repository about this discrepancy and ask them to keep `setup.py` and `requirements.txt` at par with each-other.

@dopplershift Please let me know your thoughts on this.

:bulb: For now, your last two suggestions have been implemented with those lines being commented out. I will clean them up, finally before asking you to merge them.

Reply:
I usually opt for the minimum, since at worst the missing dependency just results in an error message. I do understand the desire for caution so it's up to you as the recipe maintainer.

Reply:
Fair point. On a second thought, I prefer to leave it with a commented note on exclusion of pillow (just to inform my future-self or other future maintainers, why pillow was not necessary).

Here is the link to what I asked the package maintainer: https://github.com/jacobgil/pytorch-grad-cam/issues/180#issuecomment-997151689

![image](https://user-images.githubusercontent.com/10201242/146631289-8c4adafe-dcd4-477b-a5ef-971e8cc9ceed.png)

Reply:
@dopplershift It looks like packaging the library works just fine with the following lines commented out. Thank you, Ryan for pointing this out. I am keeping it to the bare minimum.

```yaml
# - pillow
# - numpy
# - typing-extensions ==3.7.4.3
```